src/main/java/core/basesyntax/impl/StorageImpl.java

```diff
@@ -3,17 +3,38 @@
 import core.basesyntax.Storage;
 
 public class StorageImpl<K, V> implements Storage<K, V> {
+    private static final int MAXIMUM_SIZE = 10;
+    private final K[] key;
+    private final V[] value;
+    private int size;
+
+    public @SuppressWarnings("unchecked") StorageImpl() {
```

Comment:
@SuppressWarnings("unchecked") - is redundant
```suggestion
    public StorageImpl() {
```

Reply:
agree